core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -3367,6 +3367,15 @@ public Computer createComputer() {
         return new Hudson.MasterComputer();
     }
 
+    @Override
+    public void load() {
+        try {
+            loadConfig();
+        } catch (IOException e) {
```

Comment:
Why not throw it up? The interface allows it to be. Then internal calls to `loadConfig` could simply be replaced with `load`.